src/lib/realmdb/RealmDB.js

```diff
@@ -292,23 +292,6 @@ class RealmDB implements DB, ProfileDB {
     log.debug('getFeedPage result:', numResults, offset, res.length, res)
     return res
   }
-
-  //TODO:  make sure profile contains walletaddress or enforce it in schema in realmdb
-  setProfile(profile) {
-    this.Profiles.updateOne({ user_id: this.user.id }, { user_id: this.user.id, ...profile }, { upsert: true })
-  }
-
-  getProfile() {
-    return this.Profiles.findOne({ user_id: this.user.id })
-  }
-
-  getProfileByWalletAddress(walletAddress: string) {
-    return this.Profiles.findOne({ walletAddress })
-  }
-
-  setProfileFields(fields: { key: String, field: ProfileField }) {
-    return this.Profiles.updateOne({ user_id: this.user.id }, { $set: fields })
-  }
 }
```

Comment:
this are database specific helpers that every database will need to implement.
these should be defined also in a ProfileDB interface